extensions/php/languages/php/runnables.scm

```diff
@@ -0,0 +1,54 @@
+(
+    ; Class that follow the naming convention of PHPUnit test classes
+    ; and that doesn't have the abstract modifier
+    ; and have a method that follow the naming convention of PHPUnit test methods
+    ; and the method is public
+    (class_declaration
+        modifier: (_)? @modifier
+        (#not-eq? @modifier "abstract")
+        name: (_) @name
+        (#match? @name ".*Test")
+        body: (declaration_list
+            (method_declaration
+                (visibility_modifier)? @visibility
+                (#eq? @visibility "public")
+                name: (_) @run
+                (#match? @run "test.*")
+            )
+        )
+    )
+) @phpunit-test
+
+; Class that follow the naming convention of PHPUnit test classes
+; and that doesn't have the abstract modifier
+; and have a method that has the @test annotation
+; and the method is public
+(
+    (class_declaration
+        modifier: (_)? @modifier
+        (#not-eq? @modifier "abstract")
+        name: (_) @name
+        (#match? @name ".*Test")
+        body: (declaration_list
+            ((comment) @comment
+                (#match? @comment ".*@test.*")
+            .
+            (method_declaration
+                (visibility_modifier)? @visibility
+                (#eq? @visibility "public")
+                name: (_) @run
```

Comment:
@osiewicz Is there away to match the method name and ignore if its start with test? Because we now get the following error. 

```log
[2024-05-08T10:07:18+02:00 ERROR editor] multiple different run targets found on a single line, only the last target will be rendered
```

This happends if your method looks like this:

```php
    /**
     * @test
     */
    public function testValidateCLass()
    {
        $this->assertTrue(true);
    }
```

Reply:
Can't we just modify the regex on line 31 to be `(#match? @name ".+Test")`?

Reply:
How about
```suggestion
                name: (_) @run
                (#not-match? @run "test.*")
```
?

Reply:
That works😅

Reply:
Tbh though, it looks like something we should fix on Zed side and not here. I can't really squeeze it into todays Preview, but it would be a bogus limitation if we could only render tests from a single framework/line.

I'd say: let's proceed with that `not-match` approach and later on get rid of it once we can support multiple tests on the same line.

Reply:
Yeah, that's what I thought. It looks good to me now, will make a follow-up to support pest there are a little bit more queries to support all the pest conventions.